charts/trusted-artifact-signer/values.yaml

```diff
@@ -15,6 +15,8 @@ configs:
       pullPolicy: IfNotPresent
     rolebindings:
       - segment-backup-job
+    name: segment-backup-job
+    namespace: trusted-artifact-signer-monitoring
```

Comment:
@osmman I don't know why but the namespace_create & enabled set to false should result in that ns not being required, yet the e2e is failing bc the ns doesn't exist 🤷‍♀️

Reply:
yay this was fixed with[ #125](https://github.com/securesign/sigstore-ocp/pull/125)